### ISSUE TITLE
Schema for Microsoft.Network/virtualNetworks/virtualNetworkPeerings includes the read-only peeringState property

### ISSUE THREAD

**Issue by MWL88** [Azure/azure-resource-manager-schemas#1551]
The `peeringState` property cannot be set via an ARM template deployment. Please remove this property from the ARM template schema.

**Comment by stephaniezyen** [comment]
Yes, Peering State is a read only property and as the name represents – it represents state of peering. This is needed to know the peering state which can either be connected, Disconnected or initialized.

**Comment by metaxasa** [comment]
@stephaniezyen why was this issue closed? As the `peeringState` property is read-only, it shouldn't be present in the ARM template schema.

**Comment by stephaniezyen** [comment]
@metaxasa my mistake, was in contact with the engineers on the team responsible for this schema and misunderstood the question. I have reopened the issue and am having internal discussions with them.

**Comment by metaxasa** [comment]
@stephaniezyen thank you!
